From d4e042a98a7eb559266dbbe174a7cd12a9253836 Mon Sep 17 00:00:00 2001
From: Robert Bogart <robert.bogart@joyent.com>
Date: Wed, 13 Feb 2019 19:21:05 +0000
Subject: [PATCH] MANATEE-343 manatee full rebuilds should pull from the
 primary where possible MANATEE-375 Update dev zone scripts for MANATEE-335

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index d90f925..53c1c51 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "manta": "1.2.6",
-        "manatee": "git+https://github.com/joyent/manatee.git#179a24faf5fac6c477d0459646c78c1fccf4fce8",
+        "manatee": "git+https://github.com/joyent/manatee.git#35be0bca5871d9ba35f68c75b0ec1e8aa8538f42",
         "carrier": "0.2.0"
     },
     "sdcDependencies": {
-- 
2.21.0

